src/core/accessibility/accessibilify.js

```diff
@@ -43,28 +43,42 @@ const assignEvents = (accessibleElement, button) => {
     resizeAndRepositionElement();
 };
 
-export function accessibilify(button, gameButton = true) {
+export function accessibilify(button, gameButton = true, interactive = true) {
     const sys = button.scene.sys;
     const scene = button.scene;
     const id = [scene.scene.key, button.config.id].join("__");
+    var options;
 
-    const buttonAction = () => {
-        if (!button.input.enabled) return;
-        button.emit(Phaser.Input.Events.POINTER_UP, button, sys.input.activePointer, false);
-    };
-    const mouseOver = () => button.emit(Phaser.Input.Events.POINTER_OVER, button, sys.input.activePointer, false);
-    const mouseOut = () => button.emit(Phaser.Input.Events.POINTER_OUT, button, sys.input.activePointer, false);
-
-    const options = {
-        id,
-        button,
-        class: "gel-button",
-        "aria-label": button.config.ariaLabel,
-        parent: sys.scale.parent,
-        onClick: buttonAction,
-        onMouseOver: mouseOver,
-        onMouseOut: mouseOut,
-    };
+    if (interactive === true) {
+        const buttonAction = () => {
+            if (!button.input.enabled) return;
+            button.emit(Phaser.Input.Events.POINTER_UP, button, sys.input.activePointer, false);
+        };
+        const mouseOver = () => button.emit(Phaser.Input.Events.POINTER_OVER, button, sys.input.activePointer, false);
+        const mouseOut = () => button.emit(Phaser.Input.Events.POINTER_OUT, button, sys.input.activePointer, false);
+        options = {
```

Comment:
Could we pull this out this duplicated definition of options and use a ternary? something like:
```
const options = {
            id,
            button,
            class: "gel-button",
            "aria-label": button.config.ariaLabel,
            onClick: onClick ? onClick : () => {},
            onMouseOver:  onMouseOver ? onMouseOver : () => {},
            onMouseOut: onMouseOut ? onMouseOut : () => {},
            interactive: interactive,
            parent: interactive ? sys.scale.parent : undefined,
        };
```

Reply:
Yeah, or create a defaults object with the on mouseover partsand just do 

options = { ...defaults, ...newStuff}

?